frontend/src/layout/navigation/ProjectNotice.tsx

```diff
@@ -105,7 +105,7 @@ export function ProjectNotice(): JSX.Element | null {
             action: {
                 'data-attr': 'stop-impersonation-cta',
                 onClick: () => logout(),
-                children: 'Logout',
+                children: 'Log out',
```

Comment:
Random sneaky addition ;)

Reply:
Testing live reload and fixing grammar should always go hand in hand 😀